activities.json

```diff
@@ -1,2 +1,13 @@
 [
-]
\ No newline at end of file
+  {
+    "ciuName": null, 
+    "description": "This document describes a common data model and API for the Web of Things. TheWeb Thing Descriptionprovides a vocabulary for describing physical devices connected to the World Wide Web in a machine readable format with a default JSON encoding. TheWeb Thing REST APIandWeb Thing WebSocket APIallow a web client to access the properties of devices, request the execution of actions and subscribe to events representing a change in state. Some basicWeb Thing Typesare provided and additional types can be defined using semantic extensions with JSON-LD.In addition to this specification there is a note onWeb Thing API Protocol Bindingswhich proposes non-normative bindings of the Web Thing API to various existing IoT protocols. There is also a document describingWeb of Things Integration Patternswhich provides advice on different design patterns for integrating connected devices with the Web of Things, and where each pattern is most appropriate.", 
+    "mozBugUrl": null, 
+    "mozPosition": "participating", 
+    "mozPositionDetail": null, 
+    "mozPositionIssue": 44, 
+    "org": "W3C", 
+    "title": "Web Thing API", 
+    "url": "http://iot.mozilla.org/wot"
```

Comment:
nit: https

Reply:
Actually, https seems broken there? Can we get someone to fix that?

Reply:
This site uses github pages, which doesn't support HTTPS for custom names.  Yet.

I am told by some of my contacts at Fastly that this is a problem that they are aware of and - without making any promises - they have made some significant improvements to their HTTPS serving infrastructure recently.

Short of that sort of change, it's probably not worth fixing in the short term.

Reply:
FWIW we moved this website to Mozilla infrastructure and https://iot.mozilla.org now works.

Reply:
Fixed in e3cd7b43.